lms/envs/common.py

```diff
@@ -41,6 +41,8 @@
 # The display name of the platform to be used in templates/emails/etc.
 PLATFORM_NAME = "Your Platform Name Here"
 CC_MERCHANT_NAME = PLATFORM_NAME
+# Shows up in the platform footer, eg "(c) COPYRIGHT_YEAR"
+COPYRIGHT_YEAR = "2015"
```

Comment:
Is that the proper way to do it? That means that the copyright will continuously update, even if the site is abandoned. Who should I ask for an opinion about this... maybe @nedbat would know?